library/psa_crypto.c

```diff
@@ -3007,6 +3010,24 @@ psa_status_t psa_asymmetric_sign( psa_key_handle_t handle,
         goto exit;
     }
 
+#if defined(MBEDTLS_PSA_CRYPTO_SE_C)
+    driver = psa_get_se_driver_entry( slot->lifetime );
+    if( driver != NULL )
+    {
+        const psa_drv_se_t *drv = psa_get_se_driver_methods( driver );
+        if( drv->asymmetric == NULL ||
+            drv->asymmetric->p_sign == NULL )
+        {
+            status = PSA_ERROR_NOT_SUPPORTED;
+            goto exit;
+        }
+        status = drv->asymmetric->p_sign(
+            slot->data.se.slot_number, alg, hash, hash_length, signature,
+            signature_size, signature_length );
+        goto exit;
```

Comment:
Do we need this goto?

Reply:
No, it goes to exit at this point anyway, I'll remove it.